src/platform/backends/qemu/dnsmasq_server.cpp

```diff
@@ -17,22 +17,31 @@
 
 #include "dnsmasq_server.h"
 
+#include <multipass/logging/log.h>
 #include <multipass/utils.h>
 
+#include <fmt/format.h>
+
 #include <fstream>
 namespace mp = multipass;
+namespace mpl = multipass::logging;
 
 namespace
 {
-auto start_dnsmasq_process(const QDir& data_dir, const QString& bridge_name, const mp::IPAddress& bridge_addr,
+auto start_dnsmasq_process(const QDir& data_dir, const std::string& bridge_name, const mp::IPAddress& bridge_addr,
```

Comment:
Not sure why you changed `bridge_name` from a `QString` to a `std::string`.  It's used as a `QString` in this function.